### PR TITLE
Configure all contributors

### LINKED ISSUE #158: Add a tool to recognize contributors
As we get closer to having our MVP set up, it would be good to engage more Open Source contributors. A tool like https://allcontributors.org/ can help show gratitude to participants, both technical and non-technical ❤️ 

# Requirements

- [x] Set up a tool such as https://allcontributors.org/
- [x] Put it into the README
- [ ] Could also put it into the Org Readme (public and private)

# Dependencies

- None (Add any issue dependencies here)

### PR DESCRIPTION
**Pull request by ann-kilzer**
Fixes #158 

# What changed
- Add placeholder for All Contributors Bot (following [these instructions](https://allcontributors.org/docs/en/bot/installation))

# Testing instructions
🤔 Let's see how soon the bot starts working 🤖 
I think we might have to merge to main to see if it works.

### CODE REVIEW

**Comment by ann-kilzer** [comment]
@all-contributors please add @ermish and @theyokohamalife for ideas

**Comment by ann-kilzer** [comment]
@all-contributors please add @theyokohamalife for ideas

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,3 +25,7 @@ We are working on the V2 version of the site, so if you've volunteered in the pa
 
 1. Join the `#ui-ux-design` and `#frontend-team` channels on Slack
    🎨
+
+# Giving thanks to our volunteers 👏
+
+We use the [All Contributors Bot](https://allcontributors.org/) to celebrate various contributions to our project, from coding to design to research, translation, and more.! This project would not be possible without many talented folks collaborating. Read more about [how to use the bot here](https://allcontributors.org/docs/en/bot/usage).
```
Comment (ermish) [inline CONTRIBUTING.md:31]: Typo: !.

**Review by ermish — APPROVED** [review]
LGTM outside of the small typo

**Pull request merged**